community/STEERING-COMMITTEE.md

```diff
@@ -1,61 +1,123 @@
 # Knative Steering Committee
 
-The Knative Steering Committee (SC) defines, evolves, and defends the vision,
-values, mission, and scope of the project. _The Steering Committee is a
-work-in-progress._
+The Knative Steering Committee (KSC) is the ultimate authority for the Knative
+project, and controls all aspects of the project or delegates its authority
+to other committees.
+
+KSC has delegated technical guidance for the project to the
+[Technical Oversight Committee](TECH-OVERSIGHT-COMMITTEE.md). However, the KSC
+acts as the final escalation point for any and all decisions within the project.
 
 - [Charter](#charter)
+- [Committee Meetings](#committee-meetings)
 - [Committee Mechanics](#committee-mechanics)
 - [Committee Members](#committee-members)
+  - [Allocation of seats](#allocation-of-seats)
+- [Frequently asked questions](#frequently-asked-questions)
+- [Getting in touch](#getting-in-touch)
 
 ## Charter
 
-- Non-technical project oversight
+1. Define, evolve, and defend the vision, values, mission, and scope of the
+   project - to establish and maintain the soul of Knative.
```

Comment:
The "the soul" is too vague to quantify or clear definition here. I thing things mentioned in the start of the sentence cover the "spirit" like "vision", "values", "mission", and "scope".

I recommend removing _- to establish and maintain the soul of Knative._